test/testtheme.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 var (
-	red   = &color.RGBA{R: 255, G: 0, B: 0, A: 255}
```

Comment:
Is this change required? I'm not sure what it imacts

Reply:
With the former colour the background icon of dialogs was not visible at all. With this it is at least faintly visible.